.githooks/README.md

```diff
@@ -0,0 +1,11 @@
+# dataworks-githooks
```

Comment:
githooks shouldn't be created like this, they are a submodule.  They should be created by the `initial-commit` script when creating repos, or manually using the method in the `initial-commit` script.  Unsure why you don't have them already, perhaps the sample you used doesn't include the initial commit script? 🤔

Reply:
no... I'm following the example in Dataworks-repo-template-docker
is that the wrong place for reference or?